packages/docs/stories/Welcome.mdx

```diff
@@ -0,0 +1,83 @@
+import { Meta } from "@storybook/blocks";
+
+import { Chapter, SymbolLine } from '../src/Chapter.tsx';
+
+<Meta title="Welcome" />
+
+<img src="/synergy-s.png" height="100" style={{float: 'right', marginTop: 30, marginBottom: 30}}></img>
+
+# Welcome to the Synergy Design System
+<div style={{fontSize:"x-large"}} class="sb-unstyled">
+The centralized, living style guide for **[SICK](https://www.sick.com)** application user interfaces.
+</div>
+<div style={{fontSize: "medium"}} class="sb-unstyled">
+Synergy gives you a head start for your application. Use colors, styles, components and assets to quickly spin-up your application in development ✍ and design 🎨 at the same time.
+Build multiple apps with different frameworks sharing the same UI components.
```

Comment:
Change oder adapt to the content on mosaic `About Synergy`?

The Synergy Design System empowers teams to create brand-compliant, adaptable and accessible user-interfaces across all digital products and solutions. The design system combines design with development, generating a cohesive and aligned experience for our users.

Reply:
you're right. we probably should stick to the same wording. which do you like better?

Reply:
giving it another thought. the page on mosaic is more related to SICK (brand-compliant, digital products, etc.) The description here is less SICK related and fits imho more to the open source / github character. I'll merge it as it is. We can have another chat about it at a later time...